vmdb/app/models/configuration_organization.rb

```diff
@@ -1,6 +1,11 @@
 class ConfigurationOrganization < ActiveRecord::Base
   belongs_to :provisioning_manager
   belongs_to :parent, :class_name => 'ConfigurationOrganization'
+  has_and_belongs_to_many :configuration_profiles
 
   alias_attribute :display_name, :title
+
+  def path
+    (parent.try(:path) || []).push(self)
+  end
```

Comment:
1. can we call this `subtree`
That is what it is called for [ancestry](https://github.com/stefankroes/ancestry)

2. Does this read better to you?
For some reason the code has too many `if`s in it: `try` and `||`

```
def path
  parent ? parent.path.push(self) : [self]
end
```